python/ray/tests/test_concurrency_group.py

```diff
@@ -164,6 +164,21 @@ def f2(self):
     assert "ok" == ray.get(async_actor.f2.remote())
 
 
+def test_system_concurrency_group(ray_start_regular_shared):
```

Comment:
can you test if users provide the default concurrency group, it raises an excepetion?

```python3
# It should fail.
@ray.remote(concurrency_groups={"_ray_system": 2})
class Actor:
   ...
```

Also it seems like concurrency group has a value (to limit concurrency). What's the default for the system concurrency group? Just 1?

Reply:
I think this is fine--- it's an internal feature and we don't provide any guarnatees on how it works.